pkgs/applications/misc/omorfi/hfst.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, autoconf
+, automake
+, autoreconfHook
+, bison
+, flex
+, fetchFromGitHub
+, gettext
+, icu
+, libtool
+, stdenv
+, swig
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hfst";
```

Comment:
Just in case: https://github.com/sikmir/nur-packages/blob/master/pkgs/linguistics/hfst/default.nix